File: src/main/kotlin/pl/poznan/ue/matriculation/local/dto/ApplicantUsosIdAndPotentialDuplicateStatusDto.kt
package pl.poznan.ue.matriculation.local.dto

import pl.poznan.ue.matriculation.local.domain.enum.DuplicateStatus

data class ApplicantUsosIdAndPotentialDuplicateStatusDto(
    val usosId: Long?,
    val potentialDuplicateStatus: DuplicateStatus
)

File: src/main/webapp/src/app/model/dto/applicant-usos-id-and-potential-duplicate-status-dto.ts
import {PotentialDuplicateStatus} from '../import/potential-duplicate-status.enum';

export class ApplicantUsosIdAndPotentialDuplicateStatusDto {
  usosId: number;
  potentialDuplicateStatus: PotentialDuplicateStatus;

  constructor(usosId: number, potentialDuplicateStatus: PotentialDuplicateStatus) {
    this.usosId = usosId;
    this.potentialDuplicateStatus = potentialDuplicateStatus;
  }
}
